tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectTest.java

```diff
@@ -539,17 +537,7 @@ public void testObjectInitFunctionNegative() {
     @Test(description = "Negative test to test nillable initialization")
     public void testNillableInitialization() {
         CompileResult result = BCompileUtil.compile("test-src/object/object_nillable_init.bal");
-        Assert.assertEquals(result.getErrorCount(), 10);
-        BAssertUtil.validateError(result, 0, "cannot infer type of the object from 'Person?'", 1, 14);
-        BAssertUtil.validateError(result, 1, "cannot infer type of the object from 'Person?'", 2, 14);
-        BAssertUtil.validateError(result, 2, "cannot infer type of the object from 'Person?'", 5, 18);
-        BAssertUtil.validateError(result, 3, "cannot infer type of the object from 'Person?'", 6, 18);
-        BAssertUtil.validateError(result, 4, "cannot infer type of the object from 'Person?'", 8, 10);
-        BAssertUtil.validateError(result, 5, "cannot infer type of the object from 'Person?'", 10, 10);
-        BAssertUtil.validateError(result, 6, "cannot infer type of the object from 'Person?'", 19, 25);
-        BAssertUtil.validateError(result, 7, "cannot infer type of the object from 'Person?'", 20, 25);
-        BAssertUtil.validateError(result, 8, "cannot infer type of the object from 'Person?'", 25, 19);
-        BAssertUtil.validateError(result, 9, "cannot infer type of the object from 'Person?'", 26, 19);
+        Assert.assertEquals(result.getErrorCount(), 0);
```

Comment:
Instead of simply checking the compile error count, shall we move this to the positive test cases and get rid of this now-invalid negative test?